portfolio/src/main/java/com/google/sps/servlets/DeleteCommentServlet.java

```diff
@@ -48,4 +70,41 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     
     datastore.delete(commentKeys);
   }
+
+  /** Returns the number of comments shown entered by the user, or -1 if the 
+  comment was invalid. Min must be greater than -1 and Max must be greater than 
+  or equal to min */
+  private int getUserNum(HttpServletRequest request, String parameter, int min, int max) {
+    if (min <= -1) {
+      System.err.println("Min (" + min + ") must be greater than -1 ");
+      return -1;
```

Comment:
Instead of returning -1 in the error case, try using Java's Exception framework to throw an exception when something is not valid.